zuul.d/projects.yaml

```diff
@@ -172,3 +172,11 @@
       - ansible-ee-tests
       - publish-to-galaxy
       - publish-to-automation-hub
+
+- project:
+    name: github.com/openshift/community.okd
+    default-branch: main
+    merge-mode: squash-merge
+    templates:
+      - ansible-collections-community-okd
+      - publish-to-galaxy-3pci
```

Comment:
We don't need this line. It's already done line 17.

Reply:
updated. thanks